etc/delfin/delfin.conf

```diff
@@ -6,3 +6,6 @@ api_max_limit = 1000
 [database]
 connection = sqlite:////var/lib/delfin/delfin.sqlite
 db_backend = sqlalchemy
+
+[scheduler]
+config_path = '/delfin/scheduler_config.json'
```

Comment:
how about this file in /etc/delfin ? where we have delfin.conf also present in same location